calendar/index.js

```diff
@@ -330,6 +330,20 @@ class IRNMNCalendar extends HTMLElement {
                 dayBtn.addEventListener('click', (e) =>
                     this.handleDayClick(dayBtn),
                 );
+                dayBtn.addEventListener('mouseover', (e) => {
+                    if (this.state.checkin && !this.state.checkout) {
+                        const time = parseInt(dayBtn.dataset.time);
+                        clearHighlights(this.dayButtons, [CLASS_NAMES.inRange]);
+                        this.highlightRange(this.state.checkin, time);
+                    }
+                });
+                dayBtn.addEventListener('focus', (e) => {
```

Comment:
these listeners related to accessibility/keyboard could live in a utils/accessibility.js file